update_application.sh

```diff
@@ -2,23 +2,32 @@
 #set -e
 
 echo "running $0 on remote server"
-source /tmp/load_variables.sh
+
+source /tmp/deploy/load_variables.sh
+
+# Pull deploy scripts
+echo "updating deploy scripts"
+cd $REMOTE_SCRIPT_PATH/deploy
+git pull origin master
+
+# load the vars again in case any changed during git pull
+source $REMOTE_SCRIPT_PATH/deploy/load_variables.sh
 
 set -x
 whoami
-##TODO: stop PM2 process -- how do we know the process ID? 
-pm2 stop $REMOTE_SCRIPT_PATH/ecosystem.config.json
+service nginx stop
```

Comment:
We shouldn't need to completely stop and restart nginx for an application update. Or was there something you ran into that it needed this?